### PR TITLE
Another StreamInfo refactor

### PR DESCRIPTION
**Pull request by theychx**
An attempt to make things a bit simpler.

### CODE REVIEW

**Comment by theychx** [comment]
@skorokithakis bump

File: catt/cli.py [hunk 1]
```diff
@@ -190,60 +190,54 @@ def process_subtitle(ctx, param, value):
...
     cst, stream = setup_cast(settings["device"], video_url=video_url,
                              prep="app", controller=controller)
 
     if stream.is_local_file:
         click.echo("Casting local file %s..." % video_url)
-        click.echo("Playing %s on \"%s\"..." % (stream.video_title, cst.cc_name))
-        if subtitle is None and no_subs:
-            subtitle_url = None
-        else:
+        if not (subtitle is None and no_subs):
```
Comment (skorokithakis) [inline catt/cli.py]: Would this be clearer as `if subtitle or not no_subs`?

**Comment by skorokithakis** [comment]
Oops, sorry, I've been super busy these days :/ Reviewing now.

**Comment by skorokithakis** [comment]
By the way, I've recently started using Black for formatting and isort for sorting imports. I'll add the config in a PR, it would be nice to have some aesthetics standards.

**Comment by theychx** [comment]
I tried ```black``` recently and it completely borked ```cli.py```. Any tips would be appreciated.

**Comment by skorokithakis** [comment]
Hmm, borked how? Here's [the diff](https://www.pastery.net/wxfzej/).

File: catt/cli.py (hunk 1 above)
Comment (theychx) [reply catt/cli.py]: It would.

**Comment by skorokithakis** [comment]
Great, I have pushed the pre-commit stuff, I will beautify the code when you merge. Meanwhile, you can run `pre-commit install` on master to get ready.

**Comment by theychx** [comment]
```black``` requires py3.6. We should probably clarify that.

**Comment by skorokithakis** [comment]
Ah, damnit, I missed that :/ Will clarify, thanks.

**Pull request merged**